pkg/code-generator/schemagen/cue.go

```diff
@@ -21,10 +21,11 @@ import (
 // packages to translate protobuf definitions into OpenAPI schemas with structural constraints
 // It is modeled after the istio tool: https://github.com/istio/tools/tree/master/cmd/cue-gen
 // TODO (sam-heilbron) This tool has a known flaw that makes it unusable with Gloo Edge protos
-// 	 There are open issues to track this and a GitHub discussion with more context:
-//	 https://github.com/cuelang/cue/discussions/944. We would like to move towards this
-//	 implementation and have added it now to compare schemas generated with this implementation
-//	 to schemas generated with other implementations to ensure consistency.
+//
+//	There are open issues to track this and a GitHub discussion with more context:
+//	https://github.com/cuelang/cue/discussions/944. We would like to move towards this
```

Comment:
since we are changing this line perhaps we link to the new version https://github.com/cue-lang/cue/discussions/944

Reply:
done